python/engine/niah_verify.py

```
@@ -0,0 +1,486 @@
+                    distances: list[int], num_trials: int = 5,
+                    haystack_size: int = 8192, min_prefix: int = 512,
+                    min_suffix: int = 256, seed: int = 42,
+                    gpu_device: int = 0,
+                    coherence: bool = True) -> dict:
+    """Run full NIAH verification against a checkpoint.
+
+    Returns results dict with per-distance pass rates and trial details.
+    """
+    # Load checkpoint
+    try:
+        params, cfg, build_state = nl_hecate.load_build_checkpoint(checkpoint_path)
```

Ruff (0.15.5)

[warning] Unpacked variable `build_state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)